spec/models/preference_spec.rb

```diff
@@ -1,3 +1,19 @@
+# == Schema Information
+#
+# Table name: preferences
+#
+#  id          :bigint           not null, primary key
+#  user_id     :bigint           not null
+#  name        :string
+#  description :text
+#  restriction :boolean          default(TRUE), not null
+#  created_at  :datetime         not null
+#  updated_at  :datetime         not null
+#
+# Indexes
+#
+#  index_preferences_on_user_id  (user_id)
+#
 require 'rails_helper'
 
 RSpec.describe Preference, type: :model do
```

Comment:
What about adding some test to check validations? Check [be_valid](https://www.rubydoc.info/gems/rspec-rails/RSpec%2FRails%2FMatchers:be_valid) and [validate_presence_of](https://github.com/thoughtbot/shoulda-matchers/blob/main/lib/shoulda/matchers/active_model/validate_presence_of_matcher.rb) helpers.

Also user model has already tests implemented 😉